deploy/complete/terraform/provider.tf

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+# 
+
+terraform {
+  required_version = ">= 0.12.16"
+  # required_providers {
+  #   kubernetes = ">= 1.8"
+  #   helm = ">= 1.1.1"
+  #   oci = ">= 3.73"
+  # }
+}
+
+provider "oci" {
+  tenancy_ocid = var.tenancy_ocid
+  region       = var.region
+}
+
+# https://docs.cloud.oracle.com/en-us/iaas/Content/ContEng/Tasks/contengdownloadkubeconfigfile.htm#notes
+provider "kubernetes" {
+  load_config_file       = "false"
+  cluster_ca_certificate = base64decode(yamldecode(data.oci_containerengine_cluster_kube_config.oke_cluster_kube_config.content)["clusters"][0]["cluster"]["certificate-authority-data"])
+  host                   = yamldecode(data.oci_containerengine_cluster_kube_config.oke_cluster_kube_config.content)["clusters"][0]["cluster"]["server"]
+  exec {
+    api_version = "client.authentication.k8s.io/v1beta1" # Workaround for tf k8s provider < 1.11.1 to work with orm - yamldecode(data.oci_containerengine_cluster_kube_config.oke_cluster_kube_config.content)["users"][0]["user"]["exec"]["apiVersion"]
+    args = [yamldecode(data.oci_containerengine_cluster_kube_config.oke_cluster_kube_config.content)["users"][0]["user"]["exec"]["args"][0],
```

Comment:
is it possible to derive this using an iteration range 0-6?

Reply:
The way that OKE have the token 2.0.0 and how is supported by the ORM make things more complicated. This is actually the supported way now for ORM, and was tested with the ORM workers. I hope we be able to simplify that in the future.